### PR TITLE
Fix aux button support on Veikk A15 tablets

### LINKED ISSUE #3280: Veikk A15 has no aux button support
### Description

I have made the necessary changes to fix the issue for myself, but I don't know how to implement them in a way that would work as a PR for the project without breaking support for other Veikk tablets.
Could someone look at the changes I made in my fork and add them properly?

Fork: https://github.com/adhdandy/OpenTabletDriver

### Acknowledgements

- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

### PR DESCRIPTION
**Pull request by adhdandy**
Fixes #3280.

Proof [tablet-data.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/15000732/tablet-data.txt)

### CODE REVIEW

**Comment by jamesbt365** [comment]
Tablet data as proof doesn't verify much as it doesn't necessarily prove what the tablet is doing, could you do separate data recordings for the different functions of the tablet?

**Comment by jamesbt365** [comment]
Yes.

**Comment by adhdandy** [comment]
[aux_buttons.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/15015732/aux_buttons.txt)
[movement.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/15015733/movement.txt)
[pen_buttons.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/15015734/pen_buttons.txt)
[pen_pressure.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/15015736/pen_pressure.txt)

**Review by jamesbt365 — CHANGES_REQUESTED** [review]
I'd still prefer a different naming scheme to avoid the breaking change if any other tablets use this parser in the future.

It likely accomodates all Veikk tablets that currently use the `SkipByteTabletReportParser`.

Parser looks fine.

**Comment by jamesbt365** [comment]
`VeikkV1ReportParser` should be fine for now. We'll be improving the naming scheme of our parsers at some point.

**Comment by jamesbt365** [comment]
Is tilt out of scope?

**Comment by adhdandy** [comment]
> Is tilt out of scope?

I only have the old A15 without tilt, so I don't know how that works.

File: OpenTabletDriver.Configurations/Configurations/VEIKK/A15.json [hunk 1]
```diff
@@ -25,11 +25,13 @@
       "ProductID": 4,
       "InputReportLength": 9,
       "OutputReportLength": null,
-      "ReportParser": "OpenTabletDriver.Configurations.Parsers.SkipByteTabletReportParser",
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.Veikk.VeikkA15ReportParser",
```
[the remark this answers is not in the document]
Comment (InfinityGhost) [reply OpenTabletDriver.Configurations/Configurations/VEIKK/A15.json:28]: We can change it later if it happens to fit more tablets.

**Review by gonX — APPROVED** [review]
LGTM, any parser duplication issues can be solved later (given that its already fairly messy)

File: OpenTabletDriver.Configurations/Configurations/VEIKK/A15.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamesbt365) [reply OpenTabletDriver.Configurations/Configurations/VEIKK/A15.json:28]: Will merge as is and somebody can rename it when they add tilt or other tablets to the parser.

**Review by jamesbt365 — APPROVED** [review]

**Pull request merged**